frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -227,6 +238,17 @@ const filterByGroup = (items, filters) => {
   );
 };
 
+export const groupItems = (items, groupBy) => {
```

Comment:
This specific grouping needs to be handled by the parent component `catalog-items.jsx`